### PR TITLE
Update base-fr.yaml

### PR DESCRIPTION
**Pull request by LeGnocchi**
I added some translations for the puzzle mode and fixed some spelling and grammar errors made on previous traductions.

### CODE REVIEW

**Review by Tontonflood — APPROVED** [review]

**Comment by tobspr** [comment]
Could you check the conflicts?

**Comment by LeGnocchi** [comment]
> Could you check the conflicts?

The conflicts? What do you mean by conflicts? Anyway, I will comb through all the lines to modify once again any potential errors.

**Comment by CyberGsus** [comment]
The branch has merge conflicts, i.e there are sections of the file which can't be merged automatically with the current code. You can click the box that says "Merge conflicts" (with a red circle at its left) and github will tell you more info.

**Comment by LeGnocchi** [comment]
> The branch has merge conflicts, i.e there are sections of the file which can't be merged automatically with the current code. You can click the box that says "Merge conflicts" (with a red circle at its left) and github will tell you more info.

Ok, thank you, I will see that

**Comment by LeGnocchi** [comment]
> The branch has merge conflicts, i.e there are sections of the file which can't be merged automatically with the current code. You can click the box that says "Merge conflicts" (with a red circle at its left) and github will tell you more info.

I have been using github for a short time and am not completely comfortable with the site yet. I think I figured out how to resolve conflicts (I have a "resolve conflicts" button instead of "merge conflicts") : adding a space to the places marked in red removes the red indicators but the "mark as resolved" button remains gray and cannot be clicked...

PNG

**Comment by tobspr** [comment]
It shows you have 7 conflicts left, so you'll have to solve the conflicts first :)

**Comment by LeGnocchi** [comment]
> It shows you have 7 conflicts left, so you'll have to solve the conflicts first :)

ARGH... By left clicking on "base-fr.yaml" after making the changes, this updates the number of conflicts but the button remains blocked...
Again

**Comment by LeGnocchi** [comment]
> Cela montre qu'il vous reste 7 conflits, vous devrez donc d'abord résoudre les conflits :)

It's done! :D

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -11,6 +11,16 @@ steamPage:
         Et en plus, vous devrez aussi produire de plus en plus pour satisfaire la demande. La seule solution est de construire en plus grand ! Au début vous ne ferez que découper les formes, mais plus tard vous devrez les peindre — et pour ça vous devrez extraire et mélanger des couleurs !
 
         En achetant le jeu sur Steam, vous aurez accès à la version complète, mais vous pouvez aussi jouer à une démo sur shapez.io et vous décider ensuite !
+ <<<<<<< patch-2
```
Comment (tobspr) [inline translations/base-fr.yaml:14]: This PR is not merged properly

**Review by tobspr — CHANGES_REQUESTED** [review]

File: translations/base-fr.yaml (hunk 1 above)
Comment (LeGnocchi) [reply translations/base-fr.yaml:14]: sorry if i am annoying but i dont understand what is wrong...

Comment (DerFeldspatz) [reply translations/base-fr.yaml:14]: So what basically happened is that the lines you modified were modified in the master branch after your PR. This means they can't be merged automatically. These are called conflicts. Now it's your turn to make a manual merge.
GitHub has added some syntax to mark the conflicts for you.
<<<<<<<<    means that the conflict starts here
patch-2   marks that the following is your version of the file
=======  divides your version and the master of tobspr
>>>>>>>>> master    marks the end of the conflict
So what you have to do in order to merge it is:
- Look at both versions of the file as marked
- Decide which version is correct or better
- Delete the other version so only one is left. It doesn't matter which one or if you mixed them
- Delete all of the marks GitHub has added
I'll give you an example:

This is what it looks like now:
```
<<<<<<< patch-2
     key1: This is the correct translation
=======
   key2: This is a wrong translation and it is not indented correctly
>>>>>>> master
```
So what it needs to be in the end is this:
```
    key1: This is the correct translation
```
I hope most of it is clear to you now ✌️

Comment (LeGnocchi) [reply translations/base-fr.yaml:14]: Ok thanks you! :D

**Comment by tobspr** [comment]
Still conflicting

File: translations/base-fr.yaml (hunk 1 above)
Comment (LeGnocchi) [reply translations/base-fr.yaml]: I think it's good now...

**Comment by DerFeldspatz** [comment]
Conflicts are resolved 👌

**Comment by LeGnocchi** [comment]
> Still conflicting :/

Conflicts are resolved

**Comment by tobspr** [comment]
Thanks!

**Comment by LeGnocchi** [comment]
> Thanks!

You're welcome ! 😃
(you left the "changes required" label)

**Pull request merged**